packages/gatsby-remark-smartypants/src/index.js

```diff
@@ -1,10 +1,13 @@
 const retext = require(`retext`)
 const visit = require(`unist-util-visit`)
 const smartypants = require(`retext-smartypants`)
+const _ = require('lodash')
 
-module.exports = ({ markdownAST }) => {
+module.exports = ({ markdownAST pluginOptions }) => {
```

Comment:
And actually, I moved the cheese on you a bit. Everywhere else, pluginOptions is the second argument so I just add https://github.com/gatsbyjs/gatsby/pull/1167 to make remark plugins work the same way. So your code should now be `module.exports = ({ markdownAST }, pluginOptions }) => {`.